filecoin-proofs/src/types/porep_config.rs

```diff
@@ -14,15 +14,20 @@ use storage_proofs_porep::stacked::{StackedCircuit, StackedCompound};
 use crate::{
     constants::DefaultPieceHasher,
     parameters::public_params,
-    types::{PaddedBytesAmount, PoRepProofPartitions, SectorSize, UnpaddedBytesAmount},
+    types::{
+        HSelect, PaddedBytesAmount, PoRepProofPartitions, SectorSize, UnpaddedBytesAmount,
+        UpdateProofPartitions,
+    },
 };
 
 #[derive(Clone, Copy, Debug)]
 pub struct PoRepConfig {
     pub sector_size: SectorSize,
     pub partitions: PoRepProofPartitions,
+    pub update_partitions: UpdateProofPartitions,
```

Comment:
Why is this added to the `PoRepConfig` (same for `h_select`)? It seems to me that there should be a new config type, rather than tacking this on the existing one, as they are somewhat separate